python/dlr/tflite_model.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+import tensorflow.lite as lite
+from .api import IDLRModel
+
+
+def _get_input_and_output_names(intrp):
+    """
+    Get input and output tensor names
+
+    Parameters
+    ----------
+    intrp : :py:class:`str`tf.lite.Interpreter`
+        TFLite Interpreter
+
+    Returns
+    -------
+    input_tensor_map : Input tensor Map which maps tensor_name to tensor_id
+    output_tensor_list : Output tensor list. List element is pair (name, id)
```

Comment:
Why one is a map, but the other is list?

Reply:
changed both input_tensor_map and output_tensor_list to OrderedDict. It allows to do lookup by tensor name and it keep tensor names in the original order returned TFLite Interpreter